script.module.websocket/lib/backports/ssl_match_hostname/__init__.py

```diff
@@ -0,0 +1,204 @@
+"""The match_hostname() function from Python 3.7.0, essential when using SSL."""
+
+import sys
+import socket as _socket
+
+try:
+    # Divergence: Python-3.7+'s _ssl has this exception type but older Pythons do not
+    from _ssl import SSLCertVerificationError
+    CertificateError = SSLCertVerificationError
+except:
+    class CertificateError(ValueError):
+        pass
+
+
```

Comment:
Is this included in the websockets module or is it an external module? Please confirm as I think we have this also available as a script.module. if so please remove the backports package and add it as a dependency

Reply:
It was an external module.  I looked all the way back to the helix repo and didn't see the backports module anywhere.  Should I create the backports module and submit it too?  And if so, separately, or as part of this review?

Reply:
Silly me. For the Matrix version we don't need backports anyway.  I deleted it in 0132a7a.  We'll still need to figure out how to proceed with the Leia one though.  I'll fix the other issues in that request next.